css/dashboard.css

```diff
@@ -146,3 +178,161 @@ main .card {
   padding: 1rem;
   margin-right: 0.5rem;
 }
+
+tbody tr:not(:first-child) {
+  border-top: 1px solid #eaedf3;
+}
+
+tr th {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector "tr th" to come before selector ".thead-light th" (no-descending-specificity)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5469161)